pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1796,6 +1798,38 @@ bool GSQLBackend::searchComments(const string &pattern, int maxResults, vector<C
   return false;
 }
 
+bool GSQLBackend::getSubZones(const string &pattern, vector<std::tuple<string, string>>& result)
```

Comment:
the internal api uses "Domain" instead of "Zone", and (while there are some outliers), "list" instead of "get" if multiple results are expected.

if pattern is a domain/zone name, i think it should be called "parent_zone" and be of type DNSName, as should the returned domain names. the first part of the vector appears to be the domain_id, which should be a uint32_t not a string?